omero/developers/Server/LightAdmins.txt

```diff
@@ -0,0 +1,244 @@
+    ``OriginalFile`` instances
+
+``DeleteFile``
+    delete ``OriginalFile`` objects that are in neither the managed
+    repository nor the script repository
+
+``DeleteManagedRepo``
+    delete ``OriginalFile`` objects that are in the managed repository,
+    which is to where imported image files are uploaded
+
+``DeleteScriptRepo``
+    delete ``OriginalFile`` objects that are in the script repository,
+    which is where official scripts reside
+
+``ModifyGroup``
+    make changes to groups, such as their name or permissions level
+
+``ModifyGroupMembership``
+    make changes to who is a member or owner of which group
+
+``ModifyUser``
+    make changes to users, such as their name or institution
+
+``Sudo``
+    become another user
+
+``ReadSession``
+    read other users' session UUIDs
+
+
+Bundling restrictions
+---------------------
+
+It often makes sense to apply all but a certain bundle of restrictions
+to a user. For example, users working with other users' data may benefit
+from being able to go beyond their normal group restrictions only for
+the operations of ``WriteOwned``, ``WriteManagedRepo``,
+``WriteScriptRepo``, ``Chgrp``. A facility manager importing on behalf
+of other users may appropriately be given all but the ``Sudo``
+restriction. A human resources representative may be given all but the
+``ModifyGroup``, ``ModifyGroupMembership``, ``ModifyUser`` restrictions.
+Much depends on how personnel roles are handled in each specific
+institution.
+
+.. warning::
+
+    The ``ReadSession`` restriction should be applied to *all*
+    restricted administrators. Without that restriction members of the
+    ``system`` group can read other users' session UUIDs and join those
+    sessions as the user. In contrast, while an administrator without
+    the ``Sudo`` restriction can become other users, the security system
+    prevents their using sudo to elevate their own administrative
+    powers. An administrator cannot sudo to become ``root`` to escape
+    their restrictions. :javadoc:`A security filter
+    <ome/security/basic/LightAdminPrivilegesSecurityFilter.html>`
+    assists in enforcing ``ReadSession``.
+
+
+Working with restrictions
+-------------------------
+
+Restricted administrators
+"""""""""""""""""""""""""
+
+The :javadoc:`admin service <slice2html/omero/api/IAdmin.html>` offers
+new operations for managing restrictions on administrators.
+``createLightSystemUser`` creates a new restricted administrator.
+``getAdminPrivileges`` and ``setAdminPrivileges`` manage the
+restrictions on an existing administrator.
+
+Using ``setAdminPrivileges`` to set an empty list of privileges fills
+that user's ``Experimenter.config`` property with a ``false`` value for
+every ``AdminPrivilege`` name. That user does not thus become like a
+normal user: they retain all administrative powers not explicitly
+restricted, such as being able to read all users' images. A normal
```

Comment:
Reading it 3 times, I think I understand now. The problem is that in ``A normal user's...`` part it is not totally obvious what and who is the ``normal user``, so maybe starting more slowly like ``If you want to set restriction on a user which is neither an administrator (or a member of system group), this is possible, but kinda makes no sense, because... etc" would be a less elegant, but for me much more understandable construction.